python/src/aac/lang/active_context_lifecycle_manager.py

```diff
@@ -21,11 +30,29 @@ def get_active_context(reload_context: bool = False) -> LanguageContext:
     global ACTIVE_CONTEXT
 
     if not ACTIVE_CONTEXT or reload_context:
-        ACTIVE_CONTEXT = get_initialized_language_context()
+        ACTIVE_CONTEXT = load_language_context(__state_file_name__, reload_context)
 
     return ACTIVE_CONTEXT
 
 
+def load_language_context(file_name: str, reload_context: bool = False) -> LanguageContext:
```

Comment:
Since this function is in the active context lifecycle manager, we should be more explicit as to which context is being loaded.
```suggestion
def load_active_context(file_name: str, reload_context: bool = False) -> LanguageContext:
```

That being said, this description implies it's more general, which would suggest it should belong as a method on the language context, which would have the user follow a pattern like:
```python
my_context = LanguageContext()
my_context.load_context_from_file(file_name)
previous_state_definitions = my_context.definitions
```

Reply:
Ah, good point. I'll see about moving it into the `LanguageContext` class.